charts/tempo-distributed/values.yaml

```diff
@@ -0,0 +1,125 @@
+# -- Overrides the chart's computed fullname
+fullnameOverride: tempo
+# -- Overrides the chart's computed fullname
+tempo:
+  image:
+    # -- Docker image repository
+    repository: grafana/tempo
+    # -- Overrides the image tag whose default is the chart's appVersion
+    tag: latest
+    # -- Docker image pull policy
+    pullPolicy: IfNotPresent
+  distributor:
+    replicas: 2
+    resources:
```

Comment:
Leave the resources empty with `{}`, Users should decide the resources they want to give Tempo. You could leave the suggested values as comments.